packages/react/src/elements/components/GlobalNav/GlobalNav.js

```diff
@@ -117,9 +124,16 @@ class GlobalNav extends Component {
                 {(this.props.topNav.notifications.notifications || []
                 ).map(notificationProps => (
                   <Notification
+                    unread={!this.state.readIds.includes(notificationProps.id)}
```

Comment:
:+1:. I think this easily allows for initializing this component's state with any notification IDs that have already been read, if that info is persisted and accessible from some service.